app/assets/stylesheets/govuk_admin_template/_select2_overrides.scss

```diff
@@ -0,0 +1,24 @@
+/*
+  Overrides placeholder text to gray to ensure the colour contrast
+  is accessible.
+*/
+
```

Comment:
So quite a few of the apps use the `select2` gem. I might well be being v lazy as it's not a dependency of this gem. It might better to override that directly i the apps?